client/js/components/statement/StatementExportModal.vue

```diff
@@ -154,6 +177,29 @@ export default {
   computed: {
     exportModalTitle () {
       return this.isSingleStatementExport ? Translator.trans('statement.export.do') : Translator.trans('export.statements')
+    },
+
+    exampleFileName () {
+      let exampleFileName = 'm101-jacob-meier-e5089.docx'
+      const exampleId = 'm101'
+      const exampleName = 'jacob-meier'
+      const exampleInternId = 'e5089'
```

Comment:
Nitpick: i'd suggest to chose an uppercase "M" within the value of `exampleId`, to better match the "real system".

Reply:
Yeah, I checked and in the exported name it is lower case. That's why I chose lowercase here. I think it needs to be adjusted in BE if we want to keep upper/lower case. At the moment it seems like everything is lowercase.

Reply:
Thanks for the clarification, i did not actually look into the exported files.